### ISSUE TITLE
Can't install 32-bit/64-bit PowerShell together when using MSI installer

### ISSUE THREAD

**Issue by stknohg** [PowerShell/PowerShell#5597]
Steps to reproduce
------------------

I installed 64-bit PowerShell 6.0-rc using MSI installer first.
Next I installed 32-bit PowerShell 6.0-rc using MSI installer.

Expected behavior
-----------------

Both 64-bit and 32-bit PowerShell is installed.

Actual behavior
---------------

64-bit PowerShell was removed.

Environment data
----------------

* 64-bit Windows 10 Pro(1703)

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.0-rc.2
PSEdition                      Core
GitCommitId                    v6.0.0-rc.2
OS                             Microsoft Windows 10.0.15063
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by bergmeister** [comment]
I don't have a dev machine this weekend but I think the `-arch x64` argument [here](https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L2167) is the culprit since it apparently overrides the `Platform` originally define by `ProductTargetArchitecture`. Therefore WiX wrongly thinks that the 32 bit installer is 64 bit. @SteveL-MSFT Does this need to be fixed for the RC?

**Comment by TravisEz13** [comment]
Yes, Thanks for reporting this bug.

**Comment by bergmeister** [comment]
@TravisEz13 @SteveL-MSFT  I think we can close the issue now with the PR being merged into master and the release branch.

**Comment by markekraus** [comment]
@bergmeister in the future, you can have issues auto-close on merge with special key words: https://help.github.com/articles/closing-issues-using-keywords/

**Comment by bergmeister** [comment]
Thanks, I did not know that. But it also sounds dangerous because it should be a conscious decision and review of maintainers. What if the PR says 'partially fixes '? GitHub would close the issue then. VSTS has a checkbox to close related task that can be ticked upon completion of a PR.

**Comment by markekraus** [comment]
@bergmeister It is something the PR assignee is responsible for checking before merging. Collaborators/maintainers/owners can update the PR details if needed before merging or they can re-open the issue. Issue watchers also get notified on close and can always request it be re-opened. I'd say the risk is low.

**Comment by iSazonov** [comment]
😄 There was an extra word in "Fix bug #5597"

**Comment by stknohg** [comment]
I confirmed that the problem was solved.
Thank you everyone.
